packages/generate-en10168-pdf-template/src/lib/createValidation.ts

```diff
@@ -49,7 +49,7 @@ export function createValidation(
     },
   ];
 
-  if (validation.Z04) {
+  if (validation.Z04 && typeof validation.Z04 !== 'string') {
```

Comment:
i guess the case where Z04 is a string emerged with the shared schema-definitions ?